### DISCUSSION TITLE
Fonts for Matrix Rain

### DISCUSSION THREAD

**Discussion by MrAnarchist** [rocksdanister/lively#1152] [Q&A] [answered]
Hi!
Please tell me which fonts are involved in the Matrix Rain wallpaper. I was cleaning unnecessary fonts the other day and it looks like I accidentally deleted the ones that are used in wallpaper :(

**Comment by rocksdanister** [comment, marked as answer]
`var konkani  = "゠アイウエオカキクケコサシスセソタチツテトナニヌネノハヒフヘホマミムメモヤユヨラリルレワヰヱヲンヺ・ーヽヿ0123456789"`
